style.css

```diff
@@ -503,6 +521,17 @@ h1 {
     align-self: self-start;
   }
 
+  .name-group {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    gap: 15px;
+  }
+
+  #fullName {
+    display: none;
+  }
+
   #card-container {
```

Comment:
- [ ] While you make a great effort 😃 I see that you forgot to add the ``` hr ``` line before the social links, so it will be better if you add the hr line to make it like the Figma Design as you see in the Figma there is an hr line before the social links, so let's add it like the Figma Design and make the design nice 😊👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/75eed476-66bf-4267-85ff-d01f946524ec)    |         ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/1cfc0230-8af2-4979-bab3-0754051c9ece)   |